src/windows.jl

```diff
@@ -28,17 +28,21 @@ function WindowsPath(segments::Tuple; root="", drive="", separator="\\")
     return WindowsPath(segments, root, drive, separator)
 end
 
-function WindowsPath(str::AbstractString)
+WindowsPath(str::AbstractString) = parse(WindowsPath, str)
+
+function Base.tryparse(::Type{WindowsPath}, str::AbstractString; debug=false)
```

Comment:
In this case i donot see the need to have `debug` and the use of `@debug` level.
One or the other is enoughm since on need to opt in to the `@debug` level anyway

Reply:
Yeah, I guess so.